demos/common/stream_client/stream_client.py

```diff
@@ -180,12 +199,15 @@ def start(self, *, ovms_address : str, input_name : str, model_name : str, datat
             Limits how long client could run
         limit_frames : int
             Limits how many frames should be processed
+        streaming_api : bool
+            Use experimental streaming endpoint
         """
 
-        self.cap = cv2.VideoCapture(self.source, cv2.CAP_ANY)
+        self.cap = cv2.VideoCapture(int(self.source) if len(self.source) == 1 and self.source[0].isdigit() else self.source, cv2.CAP_ANY)
```

Comment:
I'm not quite sure what is this first digit check logic.
Add a command or document what self.source is